tests/run-pass/file_manipulation.rs

```diff
@@ -22,4 +21,10 @@ fn main() {
     // Reading until EOF should get the whole text.
     file.read_to_end(&mut contents).unwrap();
     assert_eq!(bytes, contents.as_slice());
+    // Removing file should succeed
+    remove_file(path).unwrap();
+    // Opening non-existing file should fail
+    assert!(File::open(path).is_err());
```

Comment:
To properly test errno, IMO it would be a good idea to check that this returns the right kind of error.

Reply:
https://github.com/pulls